test/benchmarks/taylor.c

```diff
@@ -0,0 +1,19 @@
+#define WASM_EXPORT __attribute__((visibility("default")))
+
+// Pretty bad Pi approximation using Taylor series
+WASM_EXPORT unsigned long long taylor(unsigned n)
+{
+    double sum = 1.0;
+    int sign = -1;
```

Comment:
Could make this double too to have sign changing op, but this way there's an int to float conversion.